src/resiliation/resiliation-reason.component.js

```diff
@@ -49,7 +49,7 @@ export default {
       self.panel = false;
       self.onChange({
         ELEMENT: self.tucResiliationReason,
-        SURVEY: self.tucResiliationReasonModel,
```

Comment:
Good find

Reply:
No it wasn't set with any value.

Reply:
It was a compliment :)

Reply:
Sorry I didn't understand due to my english level ;-)